guides/v2.3/rest/modules/inventory/check-salable-quantity.md

```diff
@@ -0,0 +1,127 @@
+---
+group: rest-api
+title: Check salable quantities
+---
+
+Magento provides several endpoints that allow you to check whether a product from a specified stock is salable and the available quantity.
+
+**Service names**
+
+```
+inventorySalesApiGetProductSalabilityV1
+inventorySalesApiIsProductSalableV1
+inventorySalesApiIsProductSalableForRequestedQtyV1
+inventorySalesApiStockResolverV1
+```
+
+**REST endpoints**
+
+```
+GET /V1/inventory/get-product-salable-quantity/:sku/:stockId
+GET /V1/inventory/is-product-salable/:sku/:stockId
+GET /V1/inventory/is-product-salable-for-requested-qty/:sku/:stockId/:requestedQty
+GET /V1/inventory/stock-resolver/:type/:code
+```
+
+**Path parameters**
+
+Name | Description | Type
+--- | --- | ---
+`sku` | The SKU of the product | String
+`stock_id` | The ID of the stock the product is assigned to | Integer
+`requestedQty` | The requested quantity of products | Integer
+`type` | The type of sales channel the stock is assigned to. For 2.3, this value must be `website`. | String
+`code` | The name of the sales channel (website) | String
```

Comment:
Isn't it the code of the sales channel, not the name?

Reply:
Yes. Corrected.